components/raftstore-v2/src/worker/tablet_gc.rs

```diff
@@ -81,7 +88,16 @@ impl<EK> Task<EK> {
     #[inline]
     pub fn prepare_destroy(tablet: EK, region_id: u64, wait_for_persisted: u64) -> Self {
         Task::PrepareDestroy {
-            tablet,
+            tablet: Either::Left(tablet),
+            region_id,
+            wait_for_persisted,
+        }
+    }
+
+    #[inline]
+    pub fn prepare_destroy_path(path: PathBuf, region_id: u64, wait_for_persisted: u64) -> Self {
```

Comment:
Where will this be useful?

Reply:
Useful for gc received snapshot. We can directly send a `prepare_destroy_path` to tablet_gc once a snapshot is received without background gc tasks.